DQMOffline/Hcal/src/CaloTowersDQMClient.cc

```diff
@@ -78,18 +78,22 @@ int CaloTowersDQMClient::CaloTowersEndjob(const std::vector<MonitorElement*> &hc
    // mean number of towers per ieta
    int nx = Ntowers_vs_ieta->getNbinsX();
    float cont;
+   float conte;
    float fev = float(nevent);
 
    for (int i = 1; i <= nx; i++) {
-      cont = Ntowers_vs_ieta -> getBinContent(i) / fev ;
+      cont  = Ntowers_vs_ieta -> getBinContent(i) / fev ;
+      conte = pow(Ntowers_vs_ieta -> getBinContent(i),0.5) / fev ;
       Ntowers_vs_ieta -> setBinContent(i,cont);
+      Ntowers_vs_ieta -> setBinError(i,conte);
```

Comment:
should you not include the error on fev?

Reply:
I am not sure if we want to do that, as ntowers and fev are strongly correlated. So, we cannot add two errors quadratically. The relval stats is typically 10k, so the error on fev is 1% which is typically negligible.